server/scripts/emails.ts

```diff
@@ -41,8 +47,83 @@ export const createNotificationEmailObject = (notification_data: IPostNotificati
     to: null,
     from: 'Commonwealth <no-reply@commonwealth.im>',
     subject: subjectLine,
+    templateId: 'd-7ccab17c309f45dab93b4290b3d396d1',
     text: `${subjectLine}. <a href='${path}'>Click here</a>`,
     html: `${subjectLine}. <a href='${path}'>Click here</a>`,
   };
   return msg;
 };
+
+// regular notification email service
+
+export const createRegularNotificationEmailObject = async (user, notifications) => {
+  let emailObjArray = [];
+  emailObjArray = await Promise.all(notifications.map(async (n) => {
+    const { created_at, root_id, root_title, root_type, comment_id, comment_text,
+      chain_id, community_id, author_address, author_chain } = JSON.parse(n.notification_data);
+    const decodedTitle = decodeURIComponent(root_title).trim();
+    const nSubscription = await n.getSubscription();
+    const { category_id } = nSubscription;
+    const content = (category_id === NotificationCategories.NewComment) ? `New comment on '${decodedTitle}'`
+      : (category_id === NotificationCategories.NewMention) ? `New mention on '${decodedTitle}'`
+        : (category_id === NotificationCategories.NewReaction) ? `New reaction on '${decodedTitle}'`
+          : (category_id === NotificationCategories.NewThread) ? `New Thread in '${decodedTitle}'`
+            : (category_id === NotificationCategories.ThreadEdit) ? `'${decodedTitle}' edited`
+              : 'New notification on Commonwealth';
+    const pseudoProposal = {
+      id: root_id,
+      title: root_title,
+      chain: chain_id,
+      community: community_id,
+    };
+    const args = comment_id ? [root_type, pseudoProposal, { id: comment_id }] : [root_type, pseudoProposal];
+    const path = (getProposalUrl as any)(...args);
+    return { path, content, decodedTitle, category_id, subscription: nSubscription };
+  }));
+
+  const subject = `${notifications.length} unread notifications on Commonwealth!`;
+  const msg = {
+    to: 'zak@commonwealth.im', // TODO user.email
+    from: 'Commonwealth <no-reply@commonwealth.im>',
+    templateId: 'd-468624f3c2d7434c86ae0ed0e1d2227e',
```

Comment:
same with this, how were these generated?

Reply:
same as above